assets/styles/objects/_crop.scss

```
@@ -13,34 +13,32 @@ $crop-ratios: (
     (16:9),
 ) !default;
 
-/**
- * Provide a cropping container in order to display media (usually images)
- * cropped to certain ratios.
- *
- * 1. Set up a positioning context in which the image can sit.
- * 2. This is the crucial part: where the cropping happens.
- */
+//
+// Provide a cropping container in order to display media (usually images)
+// cropped to certain ratios.
+// 1. Set up a positioning context in which the image can sit.
```

Please add line breaks between paragraphs and lists.

```suggestion
// cropped to certain ratios.
//
// 1. Set up a positioning context in which the image can sit.
```